test_scripts/WebEngine/028_OnExitApplication_unregister.lua

```diff
@@ -0,0 +1,49 @@
+--[[ General test configuration ]]
```

Comment:
@mvorobio Please add description to PR

Example:
```
ATF Test Scripts to check #[0273](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md)

This PR is **[not ready]** for review.

### Summary
Check supporting registration for the app with WEB_VIEW hmi type 

### Dependencies:
- syncMsgVersion 6.3

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)
```

Reply:
@aderiabin Done